bundles/org.dataflowanalysis.analysis.converter/src/org/dataflowanalysis/analysis/converter/Converter.java

```diff
@@ -0,0 +1,189 @@
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+
+public class Converter {
+    private ObjectMapper objectMapper;
+    private File file;
+
+    public Converter() {
+        objectMapper = new ObjectMapper();
+    }
+
+    public CompleteDFD microToDfd(String inputFile) {
+        return microToDfd(loadMicro(inputFile));
+    }
+
+    public CompleteDFD microToDfd(MicroSecEnd inputFile) {
+        return new ProcessJSON().processMicro(inputFile);
+    }
+
+    public CompleteDFD webToDfd(String inputFile) {
+        return webToDfd(loadWeb(inputFile));
+    }
+
+    public CompleteDFD webToDfd(DFD inputFile) {
+        return new ProcessJSON().processWeb(inputFile);
+    }
+
+    public DFD dfdToWeb(String inputFile) {
+        return new ProcessDFD().parse(inputFile + ".dataflowdiagram", inputFile + ".datadictionary");
+    }
+
+    public DFD dfdToWeb(CompleteDFD complete) {
+        return new ProcessDFD().parse(complete.dataFlowDiagram(), complete.dataDictionary());
+    }
+
+    public CompleteDFD plantToDFD(String inputFile) {
+        String name = inputFile.split("\\.")[0];
+        int exitCode = runPythonScript(inputFile, "json", name + ".json");
+        if (exitCode == 0) {
+            return microToDfd(name + ".json");
+        } else {
+            System.err.println("python");
+            return null;
+        }
+
+    }
+
+    public CompleteDFD assToDFD(String inputModel, String inputFile, String modelLocation, String outputFile) {
+        final var usageModelPath = Paths.get("models", inputModel, inputFile + ".usagemodel").toString();
+        final var allocationPath = Paths.get("models", inputModel, inputFile + ".allocation").toString();
+        final var nodeCharPath = Paths.get("models", inputModel, inputFile + ".nodecharacteristics").toString();
+
+        DataFlowConfidentialityAnalysis analysis = new PCMDataFlowConfidentialityAnalysisBuilder().standalone().modelProjectName(modelLocation)
+                .usePluginActivator(Activator.class).useUsageModel(usageModelPath).useAllocationModel(allocationPath)
+                .useNodeCharacteristicsModel(nodeCharPath).build();
+
+        analysis.initializeAnalysis();
+        analysis.findAllSequences();
+        var sequences = analysis.findAllSequences();
+        var propagationResult = analysis.evaluateDataFlows(sequences);
+
+        ProcessASS ass2dfd = new ProcessASS();
+
+        return ass2dfd.transform(propagationResult);
+    }
+
+    public int runPythonScript(String in, String format, String out) {
+        String[] command = {"python3", "convert_model.py", in, format, "-op", out};
+
+        ProcessBuilder processBuilder = new ProcessBuilder(command);
+        Process process;
+        try {
+            process = processBuilder.start();
+            return process.waitFor();
+        } catch (IOException | InterruptedException e) {
+            e.printStackTrace();
```

Comment:
This prints to System.err, please replace this with an appropriate logger message